app/models/info_request.rb

```diff
@@ -744,6 +744,10 @@ def must_be_internal_or_external
     end
   end
 
+  def internally_reviewed?
```

Comment:
A better method name `internal_review_sent?` ??

Reply:
I'd go with `internal_review_requested?`. `internal_review_sent?` is also okay and says similar, but feels more about technical message delivery than user intent.

`internally_reviewed?` implies the authority _did_ a review, which may not be true.